src/libraries/System.Text.Json/docs/collection_support.md

```diff
@@ -0,0 +1,129 @@
+# JsonSerializer collection support in 3.1.x
+
+All collections (any type that derives from IEnumerable) are supported for
```

Comment:
Consider making all references to types/collections, links to the API docs for those types.
```suggestion
All collections (any type that derives from `IEnumerable`) are supported for
```